packages/bigquery/README.md

```diff
@@ -0,0 +1,89 @@
+- [Official Documentation][cloud-bigquery-docs]
+
+
+```sh
+$ npm install --save @google-cloud/bigquery
+```
+```js
+var bigquery = require('@google-cloud/bigquery')({
+  projectId: 'grape-spaceship-123',
+  keyFilename: '/path/to/keyfile.json'
+});
+
+// Access an existing dataset and table.
+var schoolsDataset = bigquery.dataset('schools');
+var schoolsTable = schoolsDataset.table('schoolsData');
+
+// Import data into a table.
+schoolsTable.import('/local/file.json', function(err, job) {});
+
+// Get results from a query job.
+var job = bigquery.job('job-id');
+
+// Use a callback.
+job.getQueryResults(function(err, rows) {});
+
+// Or get the same results as a readable stream.
+job.getQueryResults().on('data', function(row) {});
+```
+
+
+## Authentication
+
+It's incredibly easy to get authenticated and start using Google's APIs. You can set your credentials on a global basis as well as on a per-API basis. See each individual API section below to see how you can auth on a per-API-basis. This is useful if you want to use different accounts for different Google Cloud services.
+
+### On Google Compute Engine
+
+If you are running this client on Google Compute Engine, we handle authentication for you with no configuration. You just need to make sure that when you [set up the GCE instance][gce-how-to], you add the correct scopes for the APIs you want to access.
+
+``` js
+// Authenticating on a global basis.
+var projectId = process.env.GCLOUD_PROJECT; // E.g. 'grape-spaceship-123'
+
+var bigQuery = require('@google-cloud/bigquery')({
+  projectId: projectId
+});
+
+// ...you're good to go! See the next section to get started using the API.
+```
+
+### Elsewhere
+
+If you are not running this client on Google Compute Engine, you need a Google Developers service account. To create a service account:
+
+1. Visit the [Google Developers Console][dev-console].
+2. Create a new project or click on an existing project.
+3. Navigate to  **APIs & auth** > **APIs section** and turn on the following APIs (you may need to enable billing in order to use these services):
+  * BigQuery API
+4. Navigate to **APIs & auth** >  **Credentials** and then:
+  * If you want to use a new service account, click on **Create new Client ID** and select **Service account**. After the account is created, you will be prompted to download the JSON key file that the library uses to authenticate your requests.
+  * If you want to generate a new key for an existing service account, click on **Generate new JSON key** and download the JSON key file.
+
+``` js
+var projectId = process.env.GCLOUD_PROJECT; // E.g. 'grape-spaceship-123'
+
+var bigQuery = require('@google-cloud/bigquery')({
+  projectId: projectId,
+
+  // The path to your key file:
+  keyFilename: '/path/to/keyfile.json'
+
+  // Or the contents of the key file:
+  credentials: require('./path/to/keyfile.json')
+});
+
+// ...you're good to go! See the next section to get started using the APIs.
```

Comment:
I'm a little confused by this comment, it made me think there was going to be an additional section in the README talking more about the usage, but it ends after this.